composer.md

```diff
@@ -0,0 +1,33 @@
+# Composer log
+
+| Name                                 | Version      | Comment                     |
+| -------------------------------------|--------------|---------------------------- |
+| php                                  | >=5.4        | Traits                      |
+| a2lix/translation-form-bundle        | ^2.1         | Stable version start at 2.1 |
+| doctrine/doctrine-fixtures-bundle    | ^2.2         | Stable version start at 2.2 |
+| doctrine/doctrine-migrations-bundle  | ^1.1         | Framework 2.3               |
+| doctrine/orm                         | ^2.5         |                             |
+| friendsofsymfony/jsrouting-bundle    | ^1.0 or ^2.0 |                             |
+| friendsofsymfony/user-bundle         | ~2.0@dev     | 2.0 is not yet stable       |
+| friendsofvictoire/text-widget        | ^2.0         |                             |
+| friendsofvictoire/button-widget      | ^2.0         |                             |
+| incenteev/composer-parameter-handler | ^2.0         |                             |
+| jms/serializer-bundle                | ^1.0         |                             |
+| knplabs/gaufrette                    | ^0.1         |                             |
+| knplabs/doctrine-behaviors           | ^1.1         |                             |
+| knplabs/knp-menu-bundle              | ^2.1         |                             |
+| liip/imagine-bundle                  | ^1.4         |                             |
+| predis/predis                        | ^1.1         |                             |
+| sensio/distribution-bundle           | ^2.3         |                             |
+| sensio/framework-extra-bundle        | ~3.0         |                             |
+| snc/redis-bundle                     | ~2.0         |                             |
+| stof/doctrine-extensions-bundle      | ~1.2         |                             |
+| symfony/assetic-bundle               | ~2.3         |                             |
+| symfony/monolog-bundle               | ~2.4 or ~3.0 |                             |
+| symfony/symfony                      | ^2.8         |                             |
+| symfony/swiftmailer-bundle           | ^2.3         |                             |
+| troopers/alertify-bundle             | ^3.0         |                             |
```

Comment:
We could avoid to use Alertify-bundle but for now it's in every controller:

- https://github.com/Victoire/victoire/search?utf8=%E2%9C%93&q=alertify
-https://github.com/Victoire/victoire/blob/648350a6fe3ced8e615528ef4af14b91b0c1ce54/Bundle/CoreBundle/Controller/VictoireAlertifyControllerTrait.php
- https://github.com/Victoire/victoire/search?utf8=%E2%9C%93&q=%24this-%3Econgrat%28&type=Code

Should we ?